docs/content/v0.20/architecture.md

```diff
@@ -68,15 +68,20 @@ and updates *WorkStatus* objects in the ITS namespace associated with the WEC.
 
 ## KubeStellar Controller Manager
 
-This module manages binding controller and status controller. The binding controller watches `BindingPolicy` and workload objects on the Workload Definition Space (WDS) and and maintains in the Inventory and Transport Space (ITS) a wrapped object per workload object to be delivered. The status controller watches for *WorkStatus* objects on the ITS and updates the
+This module manages binding controller, transport controller, and status controller. 
```

Comment:
How does this transport controller get into this controller manager without creating a dependency cycle among go modules?

Reply:
I think this question might be biased by the knowledge of this problem. It's a technical matter that isn't necessarily related to the architecture specified here - in my opinion. 

BTW, I think the state of this doc is not very satisfying, and perhaps it requires a full re-write.

Reply:
No, what I am saying here is that the architecture document is just plain wrong. The transport controller is a distinct process in a distinct pod from a distinct `Deployment` object (I suspect) from a distinct GitHub repo. This is not a tiny detail to omit; it is a major point of the architecture.